### ISSUE TITLE
Organizing Folder Structure

### ISSUE THREAD

**Issue by Lunarnovaa** [anarchoarachnid/prefiguration-doc#1]
From an outsider's perspective, it's hard to parse what's going on in the file structure, and so it should be made more clear to someone viewing it for the first time.

# Suggestions:
- Tree like structure: Fewer folders at the top level, minimum of loose files 
- Title could be action/idea then name of the subject
- Description file within the folder to explain the contents - Link this into the CONTRIBUTING.md
- Descriptive folder titles: A new person can understand what is going to be inside a folder

**Comment by anarchoarachnid** [comment]
Done with this! We have organized the files into a readable and organized folder structure. Now we just need to fill the files with information.
